### PR TITLE
Misc. documentation updates

### LINKED ISSUE #784: Filters page shows raw code for examples after 'floor'
For the filters 'join' and onward, the [aggregated list of filters](http://shopify.github.io/liquid/filters/) is showing raw tags and unhighlighted Liquid instead of nicely formatted examples.

This page isn't super findable, so it's not terrible but it should be fixed. Maybe there's a stray `raw` tag somewhere.

### PR DESCRIPTION
**Pull request by admhlt**
This PR does a bunch of stuff:

- Bumps the GitHub Pages gem version to update all Jekyll dependencies
- Sets the site to use system fonts instead of Droid Sans
- Replaces the font-awesome icons with SVGs and removes font-awesome
- Restyles a few headings to remove underlines and use a lighter font weight
- Adds a 💧 emoji to the site title in a couple places
- Simplifies the index page, removing one heading
- Adds descriptions for all filter pages (eventually I'll do every page)
- Removes the page aggregating all the filters (it was broken anyway — fixes #784)
- Adds documentation for three filters, `compact`, `sort_natural`, and `url_decode`
- Moves `case` to the bottom of the conditionals page
- Slightly modifies `<head>` metadata for all pages, adds opengraph metadata
- Adds a favicon (the 'water-drop' icon from Shopify's internal marketing assets library)

cc @Tetsuro @fw42

After this is done, I'd like to consider moving the documentation site from the `gh-pages` branch to a `/docs` folder on the `master` branch so all work gets done on the same branch.

### CODE REVIEW

**Comment by admhlt** [comment]
Hmm. This isn't what emoji are supposed to look like on Windows.

liquid templating language - opera 2016-11-08 20 41 14

**Comment by admhlt** [comment]
I've done a few more things:

- Bumped the jQuery version and used a local file rather than calling it from a third-party CDN
- Fixed (I think) the emoji rendering on Windows — I'll verify later
- Removed the aggregate 'tags' index page, as it had the same problems as the filters page
- Added meta descriptions for all remaining pages
- Added a page discussing Liquid vs. Shopify Liquid vs. Jekyll Liquid

**Comment by admhlt** [comment]
It looks correct on Windows now:
liquid template language - opera 2016-11-13 16 29 02

File: _includes/icons/desktop-download.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Tetsuro) [inline _includes/icons/desktop-download.svg]: let's run all the SVGs through an optimizer to save them bytes. Have you got SVGO on your machine? If not you can grab it here! https://github.com/svg/svgo-gui

File: _sass/modules/_home-banner.scss [hunk 2]
```diff
@@ -2,14 +2,23 @@
   text-align: center;
   border-bottom: 1px solid lighten($color-slate, 50%);
   padding-bottom: $spacing-unit;
+  margin-bottom: $spacing-unit;
 
   h1 {
-    font-weight: bold;
+    // font-weight: bold;
```
Comment (Tetsuro) [inline _sass/modules/_home-banner.scss]: can we just delete this? :)

**Comment by Tetsuro** [comment]
Man, great stuff Adam! I love that you got rid of FontAwesome and used SVGs instead. 

I haven't had a chance to 🎩  it yet, I'll try and do that later today!

**Comment by admhlt** [comment]
Thanks bud! Appreciate it!

**Comment by pushrax** [comment]
Maybe we should have an official logo if we're going to start putting one up \*wink\* \*wink\*

**Comment by admhlt** [comment]
Heh. I'm no designer. I often wonder whether Liquid needs its own logo, but it could be kind of cute. I figured in the interim an emoji is a good way to get a sort of 'design-neutral' graphic up there.

Any thoughts on logos for Liquid, @tobi?

**Comment by admhlt** [comment]
@Tetsuro: Any opposition to me merging this this week?

**Comment by Tetsuro** [comment]
Fantastic updates man! 🎩  LGTM. LOVE the system font. 

Only one comment - to be frank I'm not too keen on the water drop icon... I wonder if we can remove it until we get a nice logo. 

If we want to keep it for now, can we vertically centre it? 

image

**Comment by tobi** [comment]
cool!

**Comment by admhlt** [comment]
@Tetsuro yeah I'm fine with getting rid of the water drop for now. We had some work done on Liquid branding last year that we might be able to dig up. I'll merge for now, minus the water drop, and we'll revisit the icon thing later.

Thanks everyone :) ❤️

**Pull request merged**